content/influxdb/cloud/organizations/bucket-schema/_index.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Manage bucket schema
+seotitle: Manage bucket schema in InfluxDB
+description: Manage bucket schema in InfluxDB using the influx CLI.
+menu:
+  influxdb_cloud:
+    name: Manage bucket schema
+    parent: Manage organizations
+weight: 105
+influxdb/cloud/tags: [buckets, bucket-schema, schema]
+---
+
+Use *bucket-schema* to restrict a bucket's columns and data types.
+
+*Bucket-schema* are measurement schema that enforce the shape of
+your bucket data.
+By default, buckets receive *schema-type* `implicit` and InfluxDB's default
+measurement schema.
+
+Buckets with schema-type `explicit` allow you to assign your own schema.
+These buckets reject write requests that do not obey their bucket-schema.
```

Comment:
```suggestion
Buckets with an `explicit` schema-type let you specify and enforce your own schema.
InfluxDB rejects write requests to buckets with an `explicit` schema-type that do not conform to the specified bucket schema.
```

Reply:
@sanderson Thanks - rewrote this intro ~20 different ways.